packages/availity-react-kit/project/app/Request/components/Patient.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import { observer } from 'mobx-react';
+import { AvField } from '../../components';
+import { Label, UncontrolledTooltip } from 'reactstrap';
+import PropTypes from 'prop-types';
+import { RequestStore } from '../stores/requestStore';
+
+@observer
+export default class Patient extends Component {
+  constructor(props) {
+    super(props);
+
+    this.updateMemberId = ::this.updateMemberId;
+  }
+
+  updateMemberId(event) {
+    const { requestStore } = this.props;
+    const { value } = event.target;
```

Comment:
I know this is only an example project, but is this destructuring really needed or is it overkill?
It could be:
```
this.props.requestStore.updateMemberId(event.target.value);
```
If you want to keep the destructuring, consider doing the event in the arguments:
```
updateMemberId({ target: { value } }) {
```

Reply:
I think it makes the code easier to read but I don't feel too strongly about it. I'll change.